content/info/about.md

```diff
@@ -1,42 +1,65 @@
+We are private academic institution that funds itself through free market.
 
-Alzymologist Oy is a dedicated laboratory for microbrewers. The services we offer are based on industrial brewing technology analytical methods, but we tune them for microbrewers needs and develop new tools - this is why we are called "Alzymologist". Microbrewers make better beer and should have better analytical tools at their disposal. And zymological knowledge should belong to people - yes, we share what we can.
+Some of us left government-funded academic institutions to stop nonsense of pleasing unaccountable Big Brother grant committees.
 
-Did we just say beer? All other zymology users are welcome - cider, mead, wine, etc.
+Some of us left conventional industry to finally start working on something that is not yet obsolete.
 
-## Our values
+Some of us left financial institutions to make economy efficient instead of robbing it.
 
-We value good beer. Of course, _consistent_ beer is something from the world of mass production, but what is really important - _consistently good_ beer - belongs to microbrewers.
+We do difficult stuff. Technological problems considered impossible, unmanageble entanglements of technical issues, tasks requiring broad vision.
 
-We value straightforwardness and openness. We tell you what we do and why, who we are and what we know.
+With many troubles surrounding modern scientific and technological world, old institutions crumbling to pieces, conflicts all around, all we can do is try to survive through diversity. If the world around falls and we still stand, we'll move on with our piece of culture - science and technology. You would do the same, right?
 
-We value community power. Working together - many microbrewers outsourcing analysis to our lab - makes our project possible. Everyone will have all the tools and expertise. And we share back - all scientific knowledge we produce ourselves we will freely share with people. Even this website is powered by [FOSS](https://www.getzola.org/).
+We are doing experimental stuff. We are The Experiment. Maybe we succeed, maybe we don't - but we'll just do our thing. Do It Ourselves.
 
-At the same time, we value intellectual property. Your data is your data, we will keep it secure. All clients data is stored and processed separately - unless you ask us otherwise. Even our email is encrypted. [Read our data protection statement](@/info/GDPR.md)
+And if we fail, we will start again.
 
+## Our values
+
+We value good beer. We value good products and good ideas. Things are meant to last, do their job, and bring joy.
+
+We value ethics, honesty, humanitarian values.
 
-We value personality. Microbrewing is based on small scale, on individuals rather than corporations. We strive to keep this spirit - so [contact us!](@/info/contacts.md)
+We value knowledge. Pursuit of knowledge. Experiments and models. Preservation of knowledge.
```

Comment:
Чет много раз слово knowledge, я бы поменял Pursuit of knowledge на curiosity, кажется это синонимы. ну и в целом кажется что знания должны быть прикладными для чего то, какой нибудь Acceleration of progress

> We value knowledge. Pursuit of knowledge. Experiments and models. Preservation of knowledge.

вот такое у меня вышло

We value knowledge. Curiosity. Experiments and models. Acceleration of progress.

Reply:
It's important to emphasize the preservation part here, maybe

```suggestion
We value knowledge. Curiosity. Experiments and models. Preservation of wisdom.
```

Reply:
thinking out loud, knowledge is about information and facts, wisdom is about knowledge application, e.g.
**Knowledge:** is knowing that tomato is a fruit
**Wisdom:** is not putting it in fruit salad 

preservation of knowledge/wisdom stands out being more abstract out of the 4 points. feel like it can be:
- **Education**: knowledge exposure and passing
- **Knowledge and experience storage**: finding a way to "preserve" those for future generations

can you elaborate further why it's important for us to preserve knowledge/wisdom?

Reply:
Because we as a civilization are losing it rapidly. This is an important topic in our projects! Some conservation idea should be relayed here.

We can't say "wisdom" for it is different thing. We can probably say "information" for as data grows information is lost and that's exactly the problem. It's ethically controversial statement though as some things should be forgotten. I'm thinking about just doing this:

```suggestion
We value wisdom. Curiosity. Experiments and models. Preservation of knowledge.
```